homeassistant/components/plugwise/climate.py

```diff
@@ -143,8 +148,12 @@ def device_state_attributes(self):
         attributes = {}
         if self._outdoor_temperature:
             attributes["outdoor_temperature"] = self._outdoor_temperature
-        attributes["available_schemas"] = self._schema_names
-        attributes["selected_schema"] = self._selected_schema
+        if self._schema_names:
+            attributes["available_schemas"] = self._schema_names
+        else:
+            attributes["available_schemas"] = "None"
```

Comment:
Just set the item when there are available schemas. Remove this.

Reply:
Sorry, I do not agree. I want to specifically show the user that there are no time-schedules/schemas available.
I hope us contributors are allowed some freedom in this?

Reply:
Our standard is to not add the item at all if there is no corresponding value.

Reply:
Our standard is to not add the item at all if there is no corresponding value.

Reply:
So there's no freedom, not even a little bit?

Reply:
Do you want me to ask for a second opinion from another member?

Reply:
Nah, after sleeping on it, I've decided I will remove the code.
I understand the thinking behind it.